src/status_im/navigation/view.cljs

```diff
@@ -71,7 +71,7 @@
            background-color            (or (get-in options [:layout :backgroundColor])
                                            (when sheet? :transparent))
            app-theme                   (rf/sub [:theme])
-           theme                       (or theme app-theme)]
+           theme                       (or app-theme theme)]
```

Comment:
screen theme should have a priority

Reply:
But if so, I think it's impossible to resolve this issue since the screen has to use app theme(light or dark).
Is there anyway to use both of them at the same time?

Reply:
you can use `(rf/sub [:theme])`

Reply:
Aha, that makes sense.